### PR TITLE
feature: ability to display source file path and line number with def…

### LINKED ISSUE #344: Support for filename and line number
The `log::Record` contains information about the filename and line number the log comes from which I have found is very useful during development.

It would be nice if it could be enabled optionally similarly to how timestamps can be turned off and module paths can be turned on.

I have a custom format-fn at the moment to do this but it would be neat if it was built-in since all other coloring and stuff is still very useful.

```log
[ LEVEL crate path/to/file.rs:1337 ]
```

### PR DESCRIPTION
**Pull request by EriKWDev**
Fixes #344 
Closes #232

### CODE REVIEW

**Comment by EriKWDev** [comment]
welp #344

File: src/fmt/mod.rs [hunk 1]
```diff
@@ -766,6 +833,8 @@
                 module_path: false,
                 target: false,
                 level: true,
+                file: false,
```
[the remark this answers is not in the document]
Comment (EriKWDev) [reply src/fmt/mod.rs]: test code behind feature, now fixed

File: src/fmt/mod.rs [hunk 2]
```diff
@@ -766,6 +833,8 @@
...
+                line_number: false,
```
[the remark this answers is not in the document]
Comment (EriKWDev) [reply src/fmt/mod.rs]: test code behind feature, now fixed

File: src/fmt/mod.rs [hunk 3]
```diff
@@ -799,6 +868,8 @@
                 module_path: true,
                 target: true,
                 level: true,
+                file: false,
```
[the remark this answers is not in the document]
Comment (EriKWDev) [reply src/fmt/mod.rs]: ack

**Comment by epage** [comment]
Thanks!

**Pull request merged**